src/django/contricleaner/lib/handlers/handler.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+from typing import List
+from contricleaner.lib.dto.list_dto import ListDTO
+
+
+class Handler(ABC):
```

Comment:
It's very common name for class. Could you please use more specific (say RowHandler, or similar)
?

Reply:
CompositeChainHandler

Reply:
In this case, I think we can name it `ListRowHandler`

Reply:
Yep, sounds good

Reply:
It can't be CompositeHandler because it doesn't relate to the Composite pattern.